Makefile.toml

```diff
@@ -395,8 +395,28 @@ fi
 
 # TODO: fix clippy lints in "sources" and check that too
 
+# For bash first-party source code
+if ! docker run --rm \
+  --network=none \
+  --user "$(id -u):$(id -g)" \
+  --security-opt label:disable \
+  -v "${BUILDSYS_TOOLS_DIR}":/tmp/tools \
+  "${BUILDSYS_SDK_IMAGE}" \
+  bash -c \
+    'cd /tmp/tools && \
+    for shell_script in $(grep -l --directories=skip --regexp="^#!.*bash$" * ); do \
+      shellcheck \
+        --external-sources \
+        --source-path=SCRIPTDIR \
+        --format=gcc \
+        --severity=warning \
+        "${shell_script}"
+    done'; then
```

Comment:
Isn't the return code from the `for` loop the result of the last `shellcheck` invocation?

I'd prefer writing this to look more like `tasks.check-fmt` or `tasks.unit-tests`, where each script is passed to a function that invokes `shellcheck` via `docker run` on just the one script argument.

Reply:
No, consistency with the other approaches was the only reason. If it's causing performance issues, I agree it's worth the cost to avoid it.